Sources/VLCFirstStepsViewController.m

```diff
@@ -48,23 +48,24 @@ - (void)viewDidLoad
     [pageVC didMoveToParentViewController:self];
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(updateTheme) name:kVLCThemeDidChangeNotification object:nil];
     [self updateTheme];
+    [self updateTitle];
     [self setupNavigationBar];
 }
 
-- (void)viewWillAppear:(BOOL)animated
+- (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection
 {
-    [super viewWillAppear:animated];
-    if ([UIDevice currentDevice].userInterfaceIdiom == UIUserInterfaceIdiomPhone) {
-        [AppUtility lockOrientation:UIInterfaceOrientationMaskPortrait];
-    }
+    [super traitCollectionDidChange:previousTraitCollection];
+    __weak typeof(self) weakSelf = self;
+
+    dispatch_async(dispatch_get_main_queue(), ^{
+        __strong typeof(weakSelf) strongSelf = weakSelf;
+        [strongSelf updateTitle];
```

Comment:
title for page is also set in `traitCollectionDidChange` method. So I need to dispatch to get correct value.